provider/go.go

```diff
@@ -13,8 +13,15 @@ import (
 type Go struct {
 	vm     *interp.Interpreter
 	script string
+	param  []TransformationConfig
+	result []TransformationConfig
```

Comment:
Not sure I understand the semantics here. Why would result be a a slice?

Reply:
Guess: symmetry and the ability to trigger multiple downstream plugins? Would it be clearer to call those in/out?

Reply:
Yes, ability to trigger multiple downstream plugins. `in`/`out` seems to be a good name, I tried different names coming from `get`/`set` over `read`/`write` to `param`/`result`. I couldn't really find a fitting wording. I'll change to `in`/`out` and we can see how that feels. :)